### ISSUE TITLE
"Google Maps JavaScript API has been loaded directly without loading=async"

### ISSUE THREAD

**Issue by mugu-fatman** [smeijer/leaflet-geosearch#393]
Hi , 
Thanks for the lovely plugin !!! 
I am using GeoSearchControl in my svelte application with google provider 

```
        const provider = new GoogleProvider({ apiKey: "" })
	const searchControl = new GeoSearchControl({
				marker: { icon: icon},
				style: "bar",
				searchLabel: "search",
				resetButton: mdiMagnify,
				provider: provider,
				resultList: resultList
			});
			map.addControl(searchControl);
			map.on("geosearch/showlocation", (result) => getCoordinates(result));
```

Started getting warning from google api  "Google Maps JavaScript API has been loaded directly without loading=async" and it has impact on the performance takes 10 - 30 secs for the map to load. 

![image](https://github.com/smeijer/leaflet-geosearch/assets/62667193/bba40718-e4e9-47d9-854d-4c052c2253ff)

Could you suggest any fixes for this ? 

Thank you

**Comment by mugu-fatman** [comment]
Found that issue is coming from this plugin 
[](https://github.com/googlemaps/js-api-loader/blob/main/src/index.ts )

not sure how to fix it

**Comment by mugu-fatman** [comment]
Please provide some suggestion

**Comment by smeijer** [comment]
I'm going to guess, but I think it originates from the library we use, `@googlemaps/js-api-loader`.

**Comment by mugu-fatman** [comment]
Issue has been fixed in the latest version v1.16.6  is there any possibility to update ? 
https://github.com/googlemaps/js-api-loader/commit/2d05217cd728175724e1fcf024e33e21cd9cc4b2

**Comment by smeijer** [comment]
Sure. Happy to merge any pull request fixing this. Please tag me in the PR.
